packages/web/src/client/styles/stitches.config.ts

```diff
@@ -89,6 +89,7 @@ const { styled, globalCss, getCssText, config, keyframes } =
     },
     media: {
       smallTabletUp: `(min-width: ${Widths.SmallTablet}px)`,
+      tabletDown: `(max-width: ${Widths.Tablet}px)`,
```

Comment:
please don't add max-width media breakpoints, its preferred to do mobile up, it's confusing when the two are mixed.